requirements.txt

```diff
@@ -10,3 +10,5 @@ glymur<1.0.0 # If you want to actually use glymur, you must also have openjpeg i
 shapely<2.0.0
 imageio<3.0.0
 pg8000<2.0.0 # required for running from the Allen Institute's internal database
+boto3
+moto
```

Comment:
is this not a test requirement?

Reply:
modified

Reply:
did you forget to add this to test_requirements?

Reply:
committed it